src/open_inwoner/cms/cases/views/status.py

```diff
@@ -383,6 +383,10 @@ def get_upload_info_context(self, case: Zaak):
                     zt_statustype_config.document_upload_description
                 )
 
+        # disable document upload message after upload
+        if self.request.session.get("uploads", ""):
+            case_type_document_upload_description = ""
+
```

Comment:
This doesn't take into account that you can view multiple cases during your session, you are now hiding the message for all cases of the user. Unset this flag after you see it has been set